src/import-data/ImportData.ts

```diff
@@ -202,14 +203,47 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
   ]);
 };
 
+interface PreparingImportMessageProps {
+  message: string;
+}
+
+export const PreparingImportOverlay = (props: PreparingImportMessageProps) => {
+  const { message } = props;
```

Comment:
Re: colors, you can get the colors for the current theme with:

```ts
const { colors } = useThemeFromContext();
```

The returned `colors` object has a method for each color in the palette:
https://github.com/DataBiosphere/terra-ui/blob/bcbeebad8dfe874d53ba8e78b1c168b597b700f1/packages/components/src/theme.ts#L5-L21

Reply:
I'm going to move the custom colors into TerraHexagonsAnimation.ts since it has a few hardcoded values that are specific to the default terra theme, and changing this code to use theme values would clash with those values. Once that's done, the hexagons animation component can be updated to use theme values.